### PR TITLE
Enhance visibility of selected splats by drawing them over everything

### PR DESCRIPTION
**Pull request by querielo**
Discussion: https://x.com/KirillD503/status/1849766725157319110?t=IoxjeScEsKT040H1b2ZUAw&s=35

This demo showcases how selected splats are rendered in the **main** branch.

[Demo in the main branch](https://github.com/user-attachments/assets/91fe0b46-1cbb-4b8e-a070-13966ef00dd2)

This demo showcases how selected splats are rendered in the **PR** branch.

[Demo PR](https://github.com/user-attachments/assets/da39429d-c4cb-40ef-abf1-29e6f7d3c41a)

If you dislike the name "underlay" I can easily change it.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,6 +52,7 @@
         "rollup-plugin-visualizer": "^5.12.0",
         "sass": "^1.79.4",
         "serve": "^14.2.3",
-        "tslib": "^2.7.0"
+        "tslib": "^2.7.0",
+        "typescript": "^5.6.3"
```
Comment (willeastcott) [inline package.json:56]: Why add the `typescript` dependency in this PR?

**Comment by willeastcott** [comment]
Congrats - I think this is the coolest PR I've seen for SuperSplat so far (partly because the implementation is so clean and concise). 👏

File: package.json (hunk 1 above)
Comment (querielo) [reply package.json:56]: If `typescript` is not globally installed, the developer will encounter the error: "Error: Cannot find package 'typescript' imported from..."

Comment (willeastcott) [reply package.json:56]: I think it's OK to leave it. 😄

**Review by willeastcott — APPROVED** [review]

**Comment by slimbuck** [comment]
Closing in favour of #251

**Pull request closed without merging**